sync/src/synchronizer/mod.rs

```diff
@@ -685,10 +691,10 @@ mod tests {
         let (chain_controller2, shared2, _notify2) = start_chain(Some(consensus.clone()), None);
         let block_number = 200;
 
-        let mut blocks: Vec<Block> = Vec::new();
+        let mut blocks: Vec<BlockView> = Vec::new();
         let mut parent = consensus.genesis_block().header().to_owned();
         for i in 1..block_number {
-            let parent_epoch = shared1.get_block_epoch(&parent.hash()).unwrap();
+            let parent_epoch = shared1.store().get_block_epoch(&parent.hash()).unwrap();
```

Comment:
Please use `shared.get_block_epoch()`, there is a bit of nuance.

Reply:
don't use ChainProvider::get_block_epoch, it will be deleted later